datalake/src/main/java/com/sequenceiq/datalake/service/sdx/attach/SdxAttachDetachUtils.java

```diff
@@ -31,22 +36,32 @@ public void updateClusterNameAndCrn(SdxCluster sdxCluster, String newName, Strin
     public void updateStack(String originalName, String newName, String newCrn) {
         String initiatorUserCrn = ThreadBasedUserCrnProvider.getUserCrn();
         ThreadBasedUserCrnProvider.doAsInternalActor(
-                regionAwareInternalCrnGeneratorFactory.iam().getInternalCrnForServiceAsString(),
-                () ->
-                stackV4Endpoint.updateNameAndCrn(
-                        0L, originalName, initiatorUserCrn, newName, newCrn
-                )
+            regionAwareInternalCrnGeneratorFactory.iam().getInternalCrnForServiceAsString(),
+            () -> {
+                try {
+                    stackV4Endpoint.updateNameAndCrn(
+                            0L, originalName, initiatorUserCrn, newName, newCrn
+                    );
+                } catch (NotFoundException e) {
+                    LOGGER.warn("Stack not found for original name: '" + originalName + "'. Skipping update of stack.");
```

Comment:
could you add `e` to the logger so we won't loose any stacktrace? same for line 62

Reply:
Yeah doing that now

Reply:
Done!